### PR TITLE
Adding robots handler

### PR DESCRIPTION
**Pull request by paganotoni**
This is related to #797

### CODE REVIEW

File: generators/assets/webpack/templates/assets/robots.txt.tmpl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+User-agent: *
```
Comment (markbates) [inline generators/assets/webpack/templates/assets/robots.txt.tmpl]: We should probably generate this file for non-webpack apps as well.

Comment (paganotoni) [reply generators/assets/webpack/templates/assets/robots.txt.tmpl]: Right, just added that default robots to the non-webpack generated app.

**Comment by paganotoni** [comment]
@markbates thoughts on this one?

**Comment by markbates** [comment]
I'm still thinking about. Not sure if this is how we should address the problem. It hasn't escaped my attention, but I need some more time to mull it over.

**Comment by markbates** [comment]
@paganotoni I'm closing this in favor of #853 which allows for a more generic serving up of the `public` folder.

**Comment by paganotoni** [comment]
Sounds good, I can see that my solution was too specific to the robots issue, thanks @markbates

**Pull request closed without merging**